crypto/evp/pmeth_lib.c

```diff
@@ -840,3 +852,11 @@ void EVP_PKEY_meth_get_param_check(const EVP_PKEY_METHOD *pmeth,
     if (*pcheck)
         *pcheck = pmeth->param_check;
 }
+
+void EVP_PKEY_meth_get_digest_custom(EVP_PKEY_METHOD *pmeth,
+                                     int (**pdigest_custom) (EVP_PKEY_CTX *ctx,
+                                                             EVP_MD_CTX *mctx))
+{
+    if (*pdigest_custom)
```

Comment:
These checks exist in other (not all???) method function getters, but are implemented differently in some.  I don't think it's a fix to do in *this* PR, but it would be good to clean this up.

In my opinion, the check should be this:

``` C
    if (pdigest_custom)
```

Reply:
> I don't think it's a fix to do in this PR,

So suggestion is to perpetuate bugs?

> but it would be good to clean this up.

??? "*Good*" to "clean up"? It's not something that would be "good to do". They are bugs that has to be fixed.

> the check should be this:

It's equivalent to demanding NULL-check in memcpy(NULL, p, 1).

Reply:
I opened an issue (#7130) to keep track of this.